pkg/cache/warm.go

```diff
@@ -157,3 +175,41 @@ func (w *Warmer) Warm(image string, opts *config.WarmerOptions) (v1.Hash, error)
 
 	return digest, nil
 }
+
+func ParseDockerfile(opts *config.WarmerOptions) ([]string, error) {
```

Comment:
Can you add some unit tests around ParseDockerfile w/ tests cases such as:
- missing Dockerfile
- invalid Dockerfile
- single stage Dockerfile
- multi-stage Dockerfile